assets/js/blocks/attribute-filter/style.scss

```diff
@@ -25,14 +16,10 @@
 				display: inline-block;
 			}
 		}
-
-		.wc-block-attribute-filter-list-count {
-			float: right;
-		}
 	}
 
-	.is-single .wc-block-attribute-filter-list-count,
-	.wc-block-dropdown-selector .wc-block-dropdown-selector__list .wc-block-attribute-filter-list-count {
+	.is-single,
+	.wc-block-dropdown-selector .wc-block-dropdown-selector__list {
```

Comment:
I brought back it to old state․ Sorry for inattentive part.

Reply:
This happen because my background is white))

![image](https://user-images.githubusercontent.com/10712881/127276238-fca65f04-6e23-4453-b7a3-0b81cd292846.png)